### PR TITLE
Import “Identifiers and Names in Kubernetes” into reference section

### PR DESCRIPTION
**Pull request by imoisharma**
Hi there, 

As a feature request, I have created a folder called `identifiers` and provided the `identifier.md` file under it. In addition, updated the `_index.md` for navigation. Please check it out and provide me with the feedback if any changes are required or not. 

Feature request details can be found [here](https://github.com/kubernetes/website/issues/29960)

Related PR can be found [here](https://github.com/kubernetes/community/pull/6137)

Looking forward to hearing from you ASAP.

Signed-off-by: Mohit Sharma <imoisharma@icloud.com>

### CODE REVIEW

**Comment by sftim** [comment]
Is there a related PR against the community repo @imoisharma? (I'd expect to see one, and we should tie reviews of the two PRs together)

**Comment by imoisharma** [comment]
Hi @sftim, I will create the Pr, and link together. Just to confirm with you, should I just removed the identifiers.md from the `community` repo, right?

**Comment by sftim** [comment]
Given https://github.com/kubernetes/community/pull/6137#issuecomment-939523974:
/retitle Import “Identifiers and Names in Kubernetes” into reference section

LGTM

File: content/en/docs/reference/identifiers/identifiers.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Identifiers and Names in Kubernetes
```
Comment (sftim) [inline content/en/docs/reference/identifiers/identifiers.md:1]: We should add [front matter](https://gohugo.io/content-management/front-matter/) here and _not_ set a title using a level-1 Markdown heading.

@imoisharma have a look at other reference documents for an example.

**Review by sftim — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/identifiers/identifiers.md (hunk 1 above)
Comment (imoisharma) [reply content/en/docs/reference/identifiers/identifiers.md]: Sorry about this, as it is my first time with this things. I have added the front-matter. Have a look :)  @sftim

File: content/en/docs/reference/_index.md [hunk 2]
```diff
@@ -94,3 +94,5 @@ An archive of the design docs for Kubernetes functionality. Good starting points
 [Kubernetes Architecture](https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md) and
 [Kubernetes Design Overview](https://git.k8s.io/community/contributors/design-proposals).
 
+## Identifiers and Names in Kubernetes
+A summarization of the goals and recommendations for identifiers in Kubernetes. More information can be found [here](/docs/reference/identifiers/identifiers/).
```
Comment (sftim) [inline content/en/docs/reference/_index.md:98]: I'd write something like this:
```markdown
Specific documents:
- [Identifiers and Names in Kubernetes](/docs/reference/identifiers/identifiers/)

There is also an archive of the design documents for Kubernetes…
```

**Review by sftim — COMMENTED** [review]
LGTM

Once this is merged we can look at cleaning it up further (for example: hyperlinks to and from https://kubernetes.io/docs/concepts/overview/working-with-objects/names/)

File: content/en/docs/reference/_index.md (hunk 2 above)
Comment (imoisharma) [reply content/en/docs/reference/_index.md:98]: Do you want me to change the index.md and provide it with your above context?

**Comment by sftim** [comment]
Hi @imoisharma

Would you be willing to squash this down to a single commit? That's what we recommend and prefer here.

File: content/en/docs/reference/_index.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/reference/_index.md:98]: Yes, I recommend making a change along the lines of what I suggested here.

Comment (imoisharma) [reply content/en/docs/reference/_index.md:98]: Sure, I'll do that

**Comment by imoisharma** [comment]
Is this looks good to you? If yes, I will push it.
> Hi @imoisharma
> 
> Would you be willing to squash this down to a single commit? That's what we recommend and prefer here.

image

**Comment by sftim** [comment]
@imoisharma I don't think I'd use the heading “Specific Document”; instead, group that link with other pages about the Kubernetes API.

File: content/en/docs/reference/_index.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/reference/_index.md:98]: If you're happy to, you could use the Markdown exactly as I wrote it.

**Comment by jimangel** [comment]
Hey @imoisharma are you still working on this? It looks like [squashing the PR](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits) and updating the title are the only things left from a review perspective!

**Comment by imoisharma** [comment]
@jimangel I'll update the PR by AEST evening time today. Cheers.

**Comment by annajung** [comment]
preview: https://deploy-preview-29993--kubernetes-io-main-staging.netlify.app/docs/reference/identifiers/identifiers/

/label tide/merge-method-squash
/lgtm

**Review by sftim — CHANGES_REQUESTED** [review]
I have some concerns here.

1. How come https://deploy-preview-29993--kubernetes-io-main-staging.netlify.app/docs/reference/identifiers/ doesn't render?
  I'd call this file `content/en/docs/reference/identifiers/identifiers.md`

2. https://kubernetes.io/docs/concepts/overview/working-with-objects/names/ already covers some of this. I'm inclined to have the reference section index link to that page (signposting) and _not_ add `content/en/docs/reference/identifiers/identifiers.md`

(we could instead move https://kubernetes.io/docs/concepts/overview/working-with-objects/names/ _into_ the reference section, and update other links accordingly)

**Comment by a-mccarthy** [comment]
@imoisharma hello, are you willing to take another look at this? looks like @sftim had a few open questions, https://github.com/kubernetes/website/pull/29993#pullrequestreview-847160549 

If you want to, you can set up a local preview of your changes. Here are some helpful docs, https://kubernetes.io/docs/contribute/new-content/open-a-pr/#preview-locally. I think the issue is related to the filename. So `content/en/docs/reference/identifiers/identifiers.md` should be `content/en/docs/reference/identifiers.md`

**Comment by jimangel** [comment]
@imoisharma hey, are you still working on this? Thanks!

**Comment by zacharysarah** [comment]
@imoisharma 👋🏻 Please feel free to `/reopen` when you're ready to continue.

/close

**Pull request closed without merging**